cpanfile

```diff
@@ -130,4 +132,5 @@ on 'develop' => sub {
     requires 'Weasel', '0.11';
     requires 'Weasel::Driver::Selenium2', '0.05';
     requires 'Weasel::Widgets::Dojo';
+    requires 'Weasel::FindExpanders';
```

Comment:
@ylavoie Where was this extra dependency introduced? I can't find it in the diff...